### PR TITLE
Add border radius to banner in Profile container

### LINKED ISSUE #1738: [Profile] implement boarder radius for info banner
Related to #1575 and #1681

Reported and discussed in https://github.com/gnosis/cowswap/issues/1714#issuecomment-953965125

I'm agree! E.g.: we can add `border-radius: 14px`
image

### PR DESCRIPTION
**Pull request by ramirotw**
# Summary

Closes #1738 

  # To Test

1. Connect wallet to a different network than mainnet
2. Go to /profile
3. Make sure the info message `Profile data is only available for mainnet. Please change the network to see it.` container has border radius set

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
Changes LGTM! 
One more related enhancement: #1757

**Comment by ramirotw** [comment]
> One more related enhancement: #1757

@alongoni can work on that issue in https://github.com/gnosis/cowswap/issues/1571

File: src/custom/pages/Profile/index.tsx [hunk 1]
```diff
@@ -174,3 +175,7 @@ const formatDecimal = (number?: number): string => {
 const formatInt = (number?: number): string => {
   return number ? number.toLocaleString() : '-'
 }
+
+const StyledNotificationBanner = styled(NotificationBanner)`
```
Comment (alongoni) [inline src/custom/pages/Profile/index.tsx:179]: Looks great!

Minor detail: Perhaps we should move the Styled component below the imports.

**Review by alongoni — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]

**Review by anxolin — APPROVED** [review]
Nice! I see it in supported networks:
image


Wondering if we also should show it in unsupported:
image

In any case, I will approve and merge this for the release.

**Comment by alfetopito** [comment]
> Wondering if we also should show it in unsupported:

I don't think it matters, there's the other warning which in my view takes precedence

**Pull request merged**